contrib/excise_stdlib.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```
For systems without `/bin/bash`